testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -412,6 +412,107 @@ QUnit.test('From renders the right types of editors according to stylingMode opt
     assert.ok($testContainer.find('.dx-field-item .dx-textbox').hasClass('dx-editor-underlined'), 'right class rendered');
 });
 
+QUnit.module('Checkbox editor value', () => {
+    QUnit.test('FormData = {}', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'boolValue', editorType: 'dxCheckBox' } ],
+            formData: { }
+        }).dxForm('instance');
+
+        assert.equal(form.getEditor('boolValue').option('value'), false);
+    });
+
+    QUnit.test('FormData = { innerObject = {} }', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'innerObject.boolValue', editorType: 'dxCheckBox' } ],
+            formData: { innerObject: { } }
+        }).dxForm('instance');
+
+        assert.equal(form.getEditor('innerObject.boolValue').option('value'), false);
+    });
+
+    QUnit.test('FormData = { boolValue: true } -> option("formData", { })', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'boolValue', editorType: 'dxCheckBox' } ],
+            formData: { boolValue: true }
+        }).dxForm('instance');
+
+        form.option('formData', { });
+        assert.equal(form.getEditor('boolValue').option('value'), false);
+    });
+
+    QUnit.test('FormData = { innerObject = { boolValue: true } } -> option("formData", { innerObject = { } })', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'innerObject.boolValue', editorType: 'dxCheckBox' } ],
+            formData: { innerObject: { boolValue: true } }
+        }).dxForm('instance');
+
+        form.option('formData', { innerObject: { } });
+        assert.equal(form.getEditor('innerObject.boolValue').option('value'), false);
+    });
+
+    [true, false, undefined].forEach(boolValue => {
+        QUnit.test(`FormData.boolValue = ${boolValue}`, function(assert) {
```

Comment:
BC?
The 'false' image was show before the changes and the 'undefiend' image is shown with these changes:

![image](https://user-images.githubusercontent.com/2094015/84399002-4b865780-ac09-11ea-81d5-abb16893605a.png)

Reply:
I have created a private option for enabling the old behavior